### PR TITLE
[GHENT-2024] Initial commit for Ghent 2024 & Organiser Summit 2024

### PR DESCRIPTION
**Pull request by yvovandoorn**
- Initial commit for Ghent
- Initial commit for Organiser Day
- Announces CFP
- CFPs are open

### CODE REVIEW

File: data/events/2024-ghent.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+description: "Devopsdays is coming to Ghent!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2024-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2024-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2024-09-04T08:00:00+02:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2024-06-05T18:00:00+02:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (glasnt) [inline data/events/2024-ghent.yml:14]:
```suggestion
enddate: 2024-09-05T18:00:00+02:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```

Comment (zipkid) [reply data/events/2024-ghent.yml:14]: Same in data/events/2024-organiser-summit.yml

Comment (yvovandoorn) [reply data/events/2024-ghent.yml:14]: this is why copy pasting is such a bad idea. will adjust!

**Comment by yvovandoorn** [comment]
Removing tag per @KrisBuytaert approval of the suggested changes with regards to redirects. 
Screenshot 2024-02-05 at 11 38 40

File: data/events/2024-ghent.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    linkedin: "https://nl.linkedin.com/in/yvovandoorn"
+  - name: "Toshaan Bharvani"
+    twitter: "toshywoshy"
+    github: "toshywoshy"
+    linkedin: "https://www.linkedin.com/in/toshaan"
+  - name: "Jos van Schouten"
+    pronouns: "he/him"
+    linkedin: "https://www.linkedin.com/in/josvanschouten/"
+    twitter: "josvanschouten"
+
```
[the remark this answers is not in the document]
Comment (zipkid) [reply data/events/2024-ghent.yml:84]: My mastodon address is @Zipkid@mastodon.social

File: data/events/2024-ghent.yml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    github: "toshywoshy"
+    linkedin: "https://www.linkedin.com/in/toshaan"
+  - name: "Jos van Schouten"
+    pronouns: "he/him"
+    linkedin: "https://www.linkedin.com/in/josvanschouten/"
+    twitter: "josvanschouten"
+  - name: "Ringo De Smet"
+    github: "ringods"
+    linkedin: "https://www.linkedin.com/in/ringodesmet/"
+    mastodon: "@ringods@hachyderm.io"
```
[the remark this answers is not in the document]
Comment (KrisBuytaert) [reply data/events/2024-ghent.yml:83]: I will work on updating the full team  later ... , yes there are people missing ...

**Comment by phrawzty** [comment]
Please sort your team membership list out, then we can merge. Cheers.

File: data/events/2024-ghent.yml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Kris Buytaert"
+    twitter: "KrisBuytaert"
+    github: "KrisBuytaert"
+    linkedin: "https://www.linkedin.com/in/krisbuytaert"
+  - name: "Rudy Gevaert"
```
[the remark this answers is not in the document]
Comment (ringods) [reply data/events/2024-ghent.yml:56]: That was indeed the last message we got from him.

**Comment by yvovandoorn** [comment]
@KrisBuytaert please include the missing team members here or Slack them to me. 

Thanks.

**Comment by ringods** [comment]
@KrisBuytaert can we please complete this PR ASAP?

**Pull request merged**